flow/packet-headers/header.yaml

```diff
@@ -123,3 +125,10 @@ components:
         macsec:
           $ref: './macsec.yaml#/components/schemas/Flow.Macsec'
           x-field-uid: 23
+        ipv6_extension_headers:
+          description: Ipv6 extension packet headers. For routing extension headers
+            the IPv6 next header is 43.
+          type: array
+          items:
+            $ref: './ipv6_routing.yaml#/components/schemas/Flow.Ipv6ExtHeaders'
```

Comment:
Object name could be `Flow.Ipv6ExtHeader` as this is an array of object.

Reply:
Updated.